scanpy/preprocessing/_simple.py

```diff
@@ -17,7 +17,7 @@
 from sklearn.utils import check_array, sparsefuncs
 
 from .. import logging as logg
-from .._compat import old_positionals
+from .._compat import old_positionals, sum
```

Comment:
`sum` is a builtin, we shouldn't be overriding those generally.

It's confusing that `sum(something)` means one thing in any python code, but means something different in this file.